lib/ansible/modules/cloud/openstack/os_port.py

```diff
@@ -78,6 +78,12 @@
    availability_zone:
      description:
        - Ignored. Present for backwards compatibility
+   vnic_type:
+     description:
+       - The type of the port tha tshould be created
```

Comment:
```suggestion
       - The type of the port that should be created
```

Reply:
Hi @mnaser ,
Thanks for the review.
Fixed.
In addition, changed the code to be cleaner.